### ISSUE TITLE
ANDI violation for Star Rating component. 

### ISSUE THREAD

**Issue by Jithinqw** [Jithinqw/wafflecss#31]
Star Rating should be focusable using tab and trigger click event on pressing enter.

**Comment by Jithinqw** [comment]
48 targeted ANDI accessibility scripts.

**Comment by Jithinqw** [comment]
Fixed in latest release v1.0.47
